### PR TITLE
Elastic delete query

### PR DESCRIPTION
**Pull request by jayaraopratik**

### CODE REVIEW

File: datastore/elastic_search/populate.py [hunk 1]
```diff
@@ -194,15 +194,20 @@ def create_dictionary_data_from_file(connection, index_name, doc_type, csv_file_
...
     data = {
-        'query': {
-            'term': {
-                'entity_data': {
-                    'value': entity_name
-                }
-            }
-        },
+        "query": {
+            "constant_score": {
```
Comment (chiragjn) [inline datastore/elastic_search/populate.py]: Is this necessary?
Minor: formatting and indentation

File: datastore/elastic_search/query.py [hunk 2]
```diff
@@ -27,13 +28,16 @@ def dictionary_query(connection, index_name, doc_type, entity_name, **kwargs):
...
     data = {
-        'query': {
-            'term': {
-                'entity_data': {
-                    'value': entity_name
-                }
-            }
-        }
+        "query": {
+            "constant_score": {
```
Comment (chiragjn) [inline datastore/elastic_search/query.py]: Same as above

File: datastore/elastic_search/populate.py (hunk 1 above)
Comment (jayaraopratik) [reply datastore/elastic_search/populate.py]: GET gogo_entity_data/_search
{
  "query": {
    "bool": {
      "must": {
        "term": {
          "entity_data": "sg_state_list_for_two_kg"
        }
      },
      "filter": {
        "term": {
          "language_script": "hi"
        }
      }
    }
  }
}

Comment (jayaraopratik) [reply datastore/elastic_search/populate.py]: Is this query more appropriate?

Comment (jayaraopratik) [reply datastore/elastic_search/populate.py]: GET gogo_entity_data/_search
{
  "query": {
    "bool": {
      "must": {
        "term": {
          "entity_data": "sg_state_list_for_two_kg"
        }
      },
      "filter": {
        "term": {
          "language_script": "hi"
        }
      }
    }
  }
}

Comment (jayaraopratik) [reply datastore/elastic_search/populate.py]: Is this query form more appropraite?

**Pull request closed without merging**